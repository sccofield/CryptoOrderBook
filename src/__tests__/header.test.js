import React from 'react';
import toJSON from 'enzyme-to-json';
import { shallow } from 'enzyme';


import Header from '../components/header';

describe('Header component', () => {
  it('should render the header component without crashing', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper.contains('Crypto Order Book')).toEqual(true);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
