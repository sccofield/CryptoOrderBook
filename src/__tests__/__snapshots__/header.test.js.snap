// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render the header component without crashing 1`] = `
<h1
  className="text-center header"
>
  Crypto Order Book
</h1>
`;
